src/DynamoCore/Search/SearchDictionary.cs

```diff
@@ -42,6 +42,15 @@ public IEnumerable<V> SearchEntries
             get { return entryDictionary.Keys; }
         }
 
+        /// <summary>
+        ///     Cleans up all recorded search entries
+        ///     Used for writing unit tests.
+        /// </summary>
+        internal void ClearAllEntries()
```

Comment:
why is this needed, SearchDictonary shouldn't be static, are there tests where you want to clear search in the middle of a test?

Reply:
I will answer in the test...where this is actually used.

Reply:
Removed